inc/saivlan.h

```diff
@@ -98,6 +98,12 @@ typedef enum _sai_vlan_attr_t
       * Value Range SAI_SWITCH_ATTR_VLAN_USER_META_DATA_RANGE */
     SAI_VLAN_ATTR_META_DATA,
 
+    /**
+      * The Multicast 802.1BR E-Channel Id (ECID), to be used in the flooding traffic
+      * sent by the CB to Port Extenders (PE). [sai_uint32_t] (CREATE_AND_SET) (default to 0)
+      */
+    SAI_VLAN_ATTR_FLOODING_ECID,
```

Comment:
@rsivasankar Sorry, mistype. “Internal External Bridge” -->> "Internal Bridge Port Extender".

An ECID is allocated within the scope the an "Internal Bridge Port Extender". Check IEEE 802.1br page 40, Figure 8-3,